cron/internal/data/gitlab-projects.csv

```diff
@@ -5499,4 +5499,14479 @@ https://gitlab.com/silenteer-oss/tutum/titan,
 https://gitlab.com/lthn.io/projects/chain/miner,
 https://gitlab.com/bz1/aiida-castep,
 https://gitlab.com/fortitudetec/elucidation,
-https://gitlab.com/mschlueter/laravel-force-https,
\ No newline at end of file
+https://gitlab.com/mschlueter/laravel-force-https,
+https://gitlab.com/highdiceroller/hdrolle,
+https://gitlab.com/jbtcd/threesixtyj,
```

Comment:
Thanks for fixing validate-projects. Do we want to do the same with `add-projects` so the list is alphabetical?

Reply:
Hm, there would be two additional csvs, and one of them may not always have changed... so seems excessive to me.